tern/analyze/docker/dockerfile.py

```diff
@@ -363,3 +364,41 @@ def split_multistage_dockerfile(dfobj):
         # dfobj_temp not empty, append to the list
         dfobj_list.append(copy.copy(dfobj_temp))
     return dfobj_list
+
+
+def get_multistage_image_dockerfiles(dfobj_multi):
```

Comment:
In this new function, we will create dockerfiles for each stage to build. A list of the dockerfile names will be returned.

Reply:
I wonder if you can make use of the `check_multistage_dockerfile` function to also return the start line and the end line of the first `FROM` line and the next `FROM` line and then use that to split the dockerfiles. I would think that for a 2 stage dockerfile, you will only need one new Dockerfile object not including the original.

Reply:
Good advice! @nishakm. I have made these changes in the new commit.